documents/sd/plantuml/application/Modules/WebBrowser/Events/ButtonClickEvent.puml

```diff
@@ -4,14 +4,18 @@ skinparam classAttributeIconSize 0
 
 class ButtonClickEvent {
     + Button : string <<get>> <<set>>
-    + URL: Uri
+    + Href : string <<get>> <<set>>
 }
 abstract class WebBrowserEvent {
-    + TabID: Guid
-    + CurrentURL: Uri
+    + TabID : int <<get>> <<set>>
+    + Deserialize(serialized:string) : void
+    + Deserialize(parsed:JsonElement) : void
+    # <<virtual>> DeserializeSpecificAttributes(JsonElement parsed) : void
```

Comment:
I'd recommend removing these protected methods from both the diagrams and the document as they are not public.